scripts/converters/to-json.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const getCategories = require('../utils/get-categories');
+const { BASE_DIR } = require('../constants');
+
+const toJSON = (doc) => {
+  const dir = path.join(BASE_DIR, ...getCategories(doc.docUrl));
+  const slug = doc.docUrl.split('/').slice(-1);
+  const fileName = `${dir}/${slug}.json`;
+
+  const json = {
+    id: doc.docId,
+    title: doc.title,
+    description: doc.shortDescription,
+    eventTypes: doc.eventTypes,
```

Comment:
I don't _think_ we're going to have JSON for any other type we migrate, but it's probably a good idea to build in some support just in case. I added a `JSON_BY_TYPE` object that holds the type-specific logic.

Reply:
💯